### PR TITLE
Clean up templates

### LINKED ISSUE #81: Improve display on RightsShell detail pages
## Is your feature request related to a problem? Please describe.
The display of individual RightsShells is pretty hard to parse.

## Describe the solution you'd like
Implement a visual hierarchy and consider whether it is necessary for all fields to appear.

### PR DESCRIPTION
**Pull request by helrond**
Makes list views more concise by adding buttons to header. Adds `__str__` method to RightsGranted. Tightens up RightsShell detail view. Fixes #81

### CODE REVIEW

**Comment by helrond** [comment]
Tests are failing here for the same reason they were failing in #82 - changes to the choices in `RIGHTS_BASIS_CHOICES` were not carried over to the tests. I've addressed that in #82, so once that is merged I'll pull in those changes here.

**Review by bonniegee — APPROVED** [review]
Looks great!

**Comment by HaSistrunk** [comment]
Cool! Two little things I noticed:
1. Maybe a nit pick, but it looks crowded at the top to my eye. What do you think about increasing the margin on the `nav` to like mb-5 instead of mb-2?
2. We need a visual indication other than color for the title and ID values in the Groupings and Rights Shells tables to show that they are links. Maybe just add an underline? I think the ID links being such a small clickable area as numbers may be something to look out for in usability testing: do folks notice them, and do they understand where those links will take them?

**Comment by helrond** [comment]
I've added additional padding to the navbar and added an underline to links. I agree that the ID is probably not the best link text (and not a big enough target). My proposal would be to wait for #80 and then use that as the link text. That may necessitate other changes in the table as well.

Additionally, we could look at a different HTML structure (divs rather than tables) so we could link the entire row. I'd suggest waiting for some user feedback first though.

**Review by HaSistrunk — APPROVED** [review]
